site/public/js/grade-report.js

```diff
@@ -0,0 +1,21 @@
+/*exported promptCustomizationUpload */
```

Comment:
Ideally this would be a new file under `site/public/mjs/` directory and loaded as a module.